packages/flutter/lib/src/services/binding.dart

```diff
@@ -216,8 +216,8 @@ mixin ServicesBinding on BindingBase, SchedulerBinding {
   /// To use a different [RestorationManager] subclasses can override
   /// [createRestorationManager], which is called to create the instance
   /// returned by this getter.
-  RestorationManager get restorationManager => _restorationManager;
-  late RestorationManager _restorationManager;
+  RestorationManager get restorationManager => _restorationManager!;
+  RestorationManager? _restorationManager;
```

Comment:
late doesn't work here with the override in flutter_test. I can't really explain it, because it doesn't make sense to me